skins/oasis/modules/LatestActivityController.class.php

```diff
@@ -58,6 +60,14 @@ public function executeIndex() {
 		$this->setVal( 'activityIcon', DesignSystemHelper::renderSvg( 'wds-icons-activity-small', 'wds-icon wds-icon-small wds-activity-icon' ) );
 		$this->setVal( 'renderCommunityEntryPoint', !empty( $wgEnableCommunityPageExt ) );
 
+		// TODO: https://wikia-inc.atlassian.net/browse/CAKE-4746
+		$currentPageTitle = $this->getContext()->getTitle()->getPrefixedText();
+		if ( $wgEnableTriviaQuizzesExt && in_array($currentPageTitle, $wgTriviaQuizzesEnabledPages) ) {
```

Comment:
```suggestion
		if ( $wgEnableTriviaQuizzesExt && in_array( $currentPageTitle, $wgTriviaQuizzesEnabledPages ) ) {
```
Just adding some extra spaces per our coding conventions.

Reply:
done in b28949d